classes/search/cmsfield.php

```diff
@@ -131,6 +133,26 @@ public function get_document($record, $options = []) {
             $valueformat = $record->valueformat;
         }
 
+        // Add mustache template to value.
+        if (!empty($defaultvalues[$record->typeid]->mustache)) {
+            $cms = new cms($cm->instance);
+            $renderer = new renderer($cms);
+            ob_start();
+            try {
+                // Indexer uses "Empty" session, it may get an error from rendering.
```

Comment:
What does this mean? In what cases "it may get an error from rendering" ?

Reply:
I tested some, and found my error was something from other plugin which was not set up properly.
I removed this try-catch section.